### DISCUSSION TITLE
请求添加集成 腾讯微云

### DISCUSSION THREAD

**Discussion by youtehub** [AlistGo/alist#2853] [Q&A] [answered]
这是腾讯微云的官方文档，感觉这个网盘使用起来也还可以，一直再用，所有在这里推荐一下
[https://open.weiyun.com/api/auth/authorize.html](https://open.weiyun.com/api/auth/authorize.html)

**Comment by anwen-anyi** [comment, marked as answer]
别想啦 已经有大佬试过 cookie问题解决不了QAQ

**Comment by youtehub** [comment]
好吧，那怪可惜的，一直觉得这个网盘挺不错的，就留着这个贴子吧，后面有人能够回看一下为何无法集成腾讯微云

**Comment by frhfeng** [comment]
等待 腾讯微云的支持。

**Comment by zhizhizz** [comment]
太可惜了，我也期待能集成腾讯微云

**Comment by guinanlin** [comment]
我还是腾讯云的很多年会员，最近1-2年， 微云估计走向没落了， 很多问题。 比如随意判定资料违反规则， 比如没法下载。真是心疼
